src/main/webapp/WEB-INF/themes/mobile/resource/css/feedback.css

```diff
@@ -1,129 +1,83 @@
-/*
- * Copyright (c) 2017 Public Library of Science
- *
- * Permission is hereby granted, free of charge, to any person obtaining a
- * copy of this software and associated documentation files (the "Software"),
- * to deal in the Software without restriction, including without limitation
- * the rights to use, copy, modify, merge, publish, distribute, sublicense,
- * and/or sell copies of the Software, and to permit persons to whom the
- * Software is furnished to do so, subject to the following conditions:
- *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
- * THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
- * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
- * DEALINGS IN THE SOFTWARE.
- */
-
 /* Feedback and Email this Article */
-#feedback-form-container, #set-grid {
-  text-align: center;
-  font-size: 1.4em;
-  width: 100%;
-  overflow-x: hidden;
+#feedback-form-container,
+#set-grid {
+  text-align: left;
+  margin: 2rem 2%;
 }
 
-#feedback-form-container form#feedbackCreate ol, #emailThisArticleForm ol {
-  margin-top: 12px;
+#feedbackCreate ol,
+#emailThisArticleForm ol {
```

Comment:
why camelCase?

Reply:
@sgatchell those were already in place.  I suspect that they are hooked up to javascript, so I didn't want to change the id names themselves.  I only removed superfluous elements and combined rules, where possible, to reduce bloat.